setup.cfg

```diff
@@ -9,6 +9,9 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
+[wheel]
+universal=1
```

Comment:
Oh okay, this section existed below. It just got reordered when I restored `[nosetests]` section back.